### ISSUE TITLE
Can't make it work

### ISSUE THREAD

**Issue by buger** [noqcks/xeol#47]
Hi there!

I'm running it like this `xeol docker.io/postgres:9 ` but it shows zero issue. 
I took it from your test suite.

What I'm doing wrong?

**UPDATE:**

BTW, Example from readme also fails: `xeol mongo:3.2`. It should be replaced by `xeol mongo:3.4`, which actually works!
So any idea why postgres does not work? 

Thanks!

**Comment by buger** [comment]
Ok, so I guess it is because of missing PURLs in endoflife db 🤔

**Comment by noqcks** [comment]
Yep, that's right. I made a PR to add an identifier

https://github.com/endoflife-date/endoflife.date/pull/2805

**Comment by noqcks** [comment]
Can you show me what you're getting when running `xeol mongo:3.2`? 

```
➜  instagram-curator git:(master) ✗ xeol mongo:3.2
 ✔ EOL DB                  [no update available]
 ✔ Parsed image
 ✔ Cataloged packages      [112 packages]
 ✔ Scanned image           [1 eol]

NAME                VERSION  EOL         DAYS EOL  TYPE
mongodb-org-server  3.2.21   2018-07-31  1713      deb
```

**Comment by buger** [comment]
I think it is M1 specific issue:
```
% docker pull mongo:3.2
3.2: Pulling from library/mongo
no matching manifest for linux/arm64/v8 in the manifest list entries
leonidbugaev@Leonids-MBP:~/go/src ▲ error 1                                                                                                                                                                                                          
% docker pull mongo:3.4
3.4: Pulling from library/mongo
Digest: sha256:d314f7eca9d3f825d091b77c40d2f3c9d6fff4de88dde2962527353476f107ba
Status: Image is up to date for mongo:3.4
docker.io/library/mongo:3.4
```

**Comment by noqcks** [comment]
interesting, shouldn't be related to architecture as im on an m1, not sure what's going on here

```
$ uname -a
Darwin MacBook-Pro-3.local 22.4.0 Darwin Kernel Version 22.4.0: Mon Mar  6 21:00:41 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T8103 arm64

$ docker pull mongo:3.2
3.2: Pulling from library/mongo
a92a4af0fb9c: Pull complete
74a2c7f3849e: Pull complete
927b52ab29bb: Pull complete
e941def14025: Pull complete
be6fce289e32: Pull complete
f6d82baac946: Pull complete
7c1a640b9ded: Pull complete
e8b2fc34c941: Pull complete
1fd822faa46a: Pull complete
61ba5f01559c: Pull complete
db344da27f9a: Pull complete
Digest: sha256:0463a91d8eff189747348c154507afc7aba045baa40e8d58d8a4c798e71001f3
Status: Downloaded newer image for mongo:3.2
docker.io/library/mongo:3.2
```

**Comment by noqcks** [comment]
this should now be fixed in version `v0.2.7` (will be released in the next hour)
